ramls/eholdings.raml

```diff
@@ -228,12 +219,6 @@ types:
   post:
     description: Create a custom package
     headers:
-      X-OKAPI-URL:
-        example: https://sampleurl.io
-      X-OKAPI-TENANT:
-        example: sample tenant
-      X-OKAPI-TOKEN:
-        example: aabbccddee
       Content-Type: 
```

Comment:
I notice that on some of these you leave the `Content-Type:` in where others you remove the whole block. What is the difference?

Reply:
awesome 👍